20FinalProject/index.html

```diff
@@ -3,8 +3,148 @@
+        <p class="top animated fadeInUp">
+          Looking for a Developer?
+        </p>
+        <p class="bottom animated fadeInUp">
+          Hire me!
+        </p>
+      </div>
+    </article>
+
+    <section class="wrapper">
+      <section class="con1">
+        <section class="a">
+
+        </section>
+        <section class="b">
+          <p id="story" class="animated bounceInRight">My Story</p>
+          <p>Graduated from Texas Tech University May 2014 with a degree in Education with concentraions in Math & History. </p>
+        </section>
+      </section>
+      <section class="con2">
+        <section class="c">
+
+        </section>
+        <section class="d">
+          <p id="education" class="animated bounceInRight">Education</p>
+          <p>ACA or the Austin Coding Academy's 10 week course taught me how to be fluent in HTML and CSS. We also learned about Java Script and Bootstrap!</p>
+        </section>
+      </section>
+      <section class="con3">
+        <section class="e">
+
+        </section>
+        <section class="f">
+          <p id="resume" class="animated bounceInRight">Resume</p>
+          <p>Having worked in Sales, service industry, and education industry I am ready to make a career change. I am ready to jump into the world of Coding and Web Development.</p>
+        </section>
+      </section>
+      <section class="con4">
+        <section class="g">
+          <p id="location" class="animated bounceInRight">Class Location</p>
+          <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3474.8795721062666!2d-98.49503568489916!3d29.432315982109213!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x865c5f519c19560d%3A0xdd83cce6523b93b9!2sCentral+Library!5e0!3m2!1sen!2sus!4v1474419323546" width="95%" height="300px" frameborder="0" style="border:0" allowfullscreen></iframe>
+        </section>
+      </section>
+      <section class="con5">
+        <section class="h">
+          <p id="contact" class="animated bounceInRight"></p>
+          <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScCHwFoo9bga4A5KJctQWSniFjQfN7S2YXOqipUcDqrS1Nz5A/viewform?embedded=true" width="1000" height="1000" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+      </section>
+
+    </section>
+
+    <div id="mycarousel" class="carousel slide">
+      <p id="portfolio">Portfolio</p>
+      <ol class="carousel-indicators">
+        <li data-target = "#mycarousel" data-slide-to="0" class="active"></li>
+        <li data-target = "#mycarousel" data-slide-to="1"></li>
+        <li data-target = "#mycarousel" data-slide-to="2"></li>
+        <li data-target = "#mycarousel" data-slide-to="3"></li>
+      </ol>
+
+      <div class="carousel-inner">
+          <div class="item active">
+            <a href="https://2104382953.github.io/intro-workbook/12LessonTwelve/in-class-project/"><img src="http://i.imgur.com/Y70OSmC.png" alt="cafe" class="img-responsive" style="width:100%;" /></a>
```

Comment:
Probably want to add `target="_blank"` to these `a` tags so the user isn't forced to leave your portfolio page.